sdks/python/apache_beam/transforms/periodicsequence.py

```diff
@@ -48,14 +48,27 @@ def initial_restriction(self, element):
   def create_tracker(self, restriction):
     return OffsetRestrictionTracker(restriction)
 
-  def restriction_size(self, unused_element, restriction):
-    return restriction.size()
+  def restriction_size(self, element, restriction):
+    return sequence_backlog_bytes(element, time.time(), restriction)
 
   # On drain, immediately stop emitting new elements
   def truncate(self, unused_element, unused_restriction):
     return None
 
 
+def sequence_backlog_bytes(element, now, offset_range):
```

Comment:
Would this be an appropriate docstring?

"Calculates size of the output that the sequence should have emitted up to now"

Reply:
Done